tests/actions/buildiso/append_line_test.py

```diff
@@ -32,6 +32,27 @@ def test_generate_system(
     )
 
 
+def test_generate_system_redhat(
+    cobbler_api, create_distro, create_profile, create_system
+):
+    # Arrange
+    test_distro = create_distro()
+    test_distro.breed = "redhat"
+    cobbler_api.add_distro(test_distro)
+    test_profile = create_profile(test_distro.name)
+    test_system = create_system(profile_name=test_profile.name)
+    blendered_data = utils.blender(cobbler_api, False, test_system)
+    test_builder = AppendLineBuilder(test_distro.name, blendered_data)
+
+    # Act
+    result = test_builder.generate_system(test_distro, test_system, False)
+
+    # Assert
+    # Very basic test yes but this is the expected result atm
+    # TODO: Make tests more sophisticated
+    assert result == f"  APPEND initrd=/{test_distro.name}.img inst.ks=default.ks"
```

Comment:
Upstream (in https://github.com/cobbler/cobbler/pull/3335/files), this file contains `APPEND initrd={...}`. We seem to add `/` after `initrd=`

Is that expected?

Reply:
Yes, that's due to the timing of the original PR and this backport. In the meantime, this was changed in both the upstream and in our code base. This is the upstream commit that brought the change: https://github.com/cobbler/cobbler/commit/67e8a2cafc